reference/api/search.mdx

```diff
@@ -1162,3 +1169,15 @@ Use a custom vector to perform a search query. Must be an array of numbers corre
 <Capsule intent="warning">
 Meilisearch will return an error if you use `vector` before activating your instance's `vectorStore` and [configuring a custom embedder](/reference/api/settings#embedders-experimental).
 </Capsule>
+
+### Display `_vectors` in response (experimental)
+
+**Parameter**: `retrieveVectors`<br />
+**Expected value**: `true` or `false`<br />
+**Default value**: `false`
+
+Return document vector data with search results.
```

Comment:
Do we want to specify the format here?

The document vector data will be injected in documents in a `_vectors` field that follows the same format as for passing the vector data to Meilisearch. Returned vector data always uses the long-form variant of the format and specifies if vectors will be regenerated on document updates.